integration-tests/deploy_R_models.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import, print_function
```

Comment:
In order to integrate this with our unit tests, it would be helpful to make use of Python's [unittest](https://docs.python.org/2/library/unittest.html) library. See [https://github.com/ucbrise/clipper/blob/develop/integration-tests/clipper_manager_tests.py](https://github.com/ucbrise/clipper/blob/develop/integration-tests/clipper_manager_tests.py) for an example. Please convert this to a Python unit test, and let me know if you have any questions.